services/spire/federation/clusterfederatedtrustdomain-cluster1.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: spire.spiffe.io/v1alpha1
+kind: ClusterFederatedTrustDomain
+metadata:
+  name: cluster2.demo
```

Comment:
Mask this value also as per the name of your cluster. Document how to retrieve the cluster name